dashboard/agent.py

```diff
@@ -304,7 +302,7 @@ async def _check_parent():
             args.redis_address, password=args.redis_password)
         traceback_str = ray.utils.format_error_message(traceback.format_exc())
         message = ("The agent on node {} failed with the following "
-                   "error:\n{}".format(os.uname()[1], traceback_str))
+                   "error:\n{}".format(platform.uname()[1], traceback_str))
```

Comment:
can we just use `socket.gethostname` here instead? I think that's what we use in other places in the code